lms/templates/courseware/courseware-chromeless.html

```diff
@@ -0,0 +1,159 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%! from django.template.defaultfilters import escapejs %>
+<%! from microsite_configuration import page_title_breadcrumbs %>
+<%! from edxnotes.helpers import is_feature_enabled as is_edxnotes_enabled %>
```

Comment:
You can put all of these in one block:

    <%!
        from django.utils.translation import ugettext as _
        from django.template.defaultfilters import escapejs
        ...
    %>